oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/nodetype/NodeTypeIndex.java

```diff
@@ -102,7 +102,10 @@ public Cursor query(Filter filter, NodeState root) {
     
     @Override
     public String getPlan(Filter filter, NodeState root) {
-        return "nodeType " + filter.toString();
+        return "nodeType\n" +
+                "    path: " + filter.getPath() + "\n" +
+                "    primaryTypes: " + filter.getPrimaryTypes().toString() + "\n" +
+                "    mixinTypes: " + filter.getMixinTypes().toString() + "\n";
```

Comment:
`toString` calls are redundant:

```suggestion
                "    primaryTypes: " + filter.getPrimaryTypes() + "\n" +
                "    mixinTypes: " + filter.getMixinTypes() + "\n";
```